markdown/retail-dev-workshop/retail-dev-workshop.md

```diff
@@ -0,0 +1,408 @@
+author: Andrew Roberts
+summary: This codelab walks you through how to use Solace PubSub+ Event Portal to design applications, events, and schemas, export AsyncAPI specifications, generate code using AsyncAPI Generator, and see your events flow across your apps.
+id: retail-dev-workshop
+tags: workshop
+categories: retail, spring, event portal, asyncapi
```

Comment:
We do not have a logo for "retail" as a category thats why in the preview link you will see no logo for your codelab. Since the icon on the codelab card on the main page is determined by the **first** element in the categories array, we have two options
1. We can add an icon for retail 
2. start with spring as the first category then list the other categories

Reply:
3. If it's not spring heavy you can also just start with "Solace" and it will get the solace logos and colors. (Note I haven't had a chance to checkout the codelab yet)